bsip-0018.md

```diff
@@ -88,104 +92,159 @@ This bug will be fixed. See
 
-## `bid_settlement_funds_operation`
+The condition can easily be checked at the time the price feed is updated.
+Obviously, this requires a price feed. Currently it is not possible to
+publish a price feed for assets that have seen global settlement. This
+restriction will be removed.
+
+## Recollateralize
 
 This applies only to SmartCoins, not to Prediction Markets.
 
+### Overview
+
 The idea of turning the settlement fund into a short position when its
 value has increased sufficiently can easily be extended. If the value of
-the settlement fund itself is not sufficient to create a sufficiently
-collateralized short position (in terms of price feed and MCR), an
-investor could volunteer to add the required amount of collateral to the
-fund and take ownership of the resulting short position
-(collateral+debt).
-
-This operation is all that is needed empty the settlement pool and re-enable price feeds.
-It has the following payload:
-
- * `fee` (asset_type): The operation requires a fee to be paid               
- * `symbol` (asset_id_typ): Symbol that has a settlement fund to be claimed.
- * `account` (account_type): This account obtains the collateral **as well** as
-   the debt (i.e. call position) and has to either pay additional collateral,
-   provide shares of the BitAsset to reduce the outstanding debt, or a combination
-   of both.
- * `additional_collateral` (asset_type): Collateral paid by the account in
-   order to support the call position
- * `obtain_settlement_funds` (asset_type): The amount of settlement funds the
-   account is willing to obtain
+the settlement fund itself is too low to create a sufficiently collateralized
+short position (in terms of price feed and MCR), investors could volunteer to
+add the required amount of collateral to the fund and take ownership of the
+resulting short position (collateral+debt).
+
+The proposed operation enables potential investors to "bid" additional
+collateral for taking over part of the debt (or all of it). When enough bids
+have been made to cover the full outstanding debt, and all of them are
+sufficiently collateralized (in terms of price feed and MCR), the
+settlement_fund and the bids are turned into call positions. Finally, the
+settlement_price is removed from the asset, which revives it.
+
+If the available bids cover more than the outstanding debt, bids with a higher
+collateral/debt ratio are preferred over those with a lower ratio. The intent is
+to turn the competition among investors into better collateralized calls, which
+is in the interest of the MPA holders.
+
+### `bid_collateral_operation`
+
+The operation has the following payload:
+
+* `fee`(asset_type): The operation requires a fee to be paid
+* `bidder`(account_type): This account pays the additional collateral and
+  will become the owner of the resulting call position, if the bid is accepted
+* `additional_collateral`(asset_type): Collateral paid by the account in order
```

Comment:
Can we call this just `collateral`. I don't think the `additional_` has any particular meaning, right?